l10n_it_central_journal/wizard/print_giornale.py

```diff
@@ -103,12 +103,14 @@ def get_line_ids(self):
             aml.date >= %(date_from)s
             AND aml.date <= %(date_to)s
             AND am.state in %(target_type)s
+            AND aml.journal_id in %(journal_ids)s
```

Comment:
> > Stampavi con l'utente admin? Perché se l'utente non è admin, dovresti ottenere #2206, puoi verificare se la fix (#2207) per v10 di quella issue risolve anche il tuo caso?
> 
> No con utente admin nessun problema. Ammetto di non aver visto prima la #2207 che mi sembra risolva lo stesso problema, però in quella PR se non erro, viene riportato il codice a come era in versioni precedenti.
> Rimuove la query SQL ed esegua la search tramite python, che era stata rimossa credo per problemi di performance.

Si passò alla query in https://github.com/OCA/l10n-italy/pull/1436, @alessandrocamilli ricordi come mai abbandonasti la ricerca tramite ORM?

Facendo la query diretta si rischia di bypassare diverse regole, prime tra tutte quelle per multi-company.
Penso sia meglio fare la ricerca tramite ORM (https://github.com/OCA/l10n-italy/pull/2207 dovrebbe risolvere il problema in descrizione), oppure ricostruire la query con una _where condition_ equivalente a quella che farebbe l'ORM in modo da applicare regole di accesso e sui record

Reply:
Anche secondo me conviene utilizzare ORM. Non credo avremo problemi di prestazioni ma se li avessimo potremo valutare altre ottimizzazioni.